scripts/GENERIC-README-PART.md

```diff
@@ -0,0 +1,27 @@
+## Support
```

Comment:
Minor: Consider renaming it to `COMMON-README.md`. "Generic" has a slightly different tone.

Reply:
yes